scripts/installDepsScripts/installRustDeps.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+set -e
+
+
+# Check if the default Rust comes from rustup, based on the official
+# recommended method of installation:
+# https://www.rust-lang.org/learn/get-started
+declare is_rustup_default=$(rustup show | awk '/\(default\)/')
+
+
+# If Rust comes from rustup, update Rust
+if [[ $is_rustup_default ]]; then
+
+    rustup update
+
+else
+    sleep 1
+    # Remove the Rust installation comming from apt instead of rustup
+    sudo apt-get remove *rust* -y
```

Comment:
Why remove these packages?

rustup will put ~/.cargo/bin at the front of your PATH.

Reply:
To remove the previous rust packages coming from the Ubuntu source installed through `apt`. To ensure that the rust packages used are always the ones coming from `rustup` and avoid the ones coming from Ubuntu being used unexpectedly, leading to unexpected behaviors. But I don't know much about Rust, so this may not make much sense for Rust. So, is it ok to keep the rust packages installed through `apt`?

Reply:
Ok, I will remove it